### ISSUE TITLE
Ubuntu 19.10 zfs boot failed

### ISSUE THREAD

**Issue by ghost** [damentz/liquorix-package#23]
hello, I was unable boot the system with installed liquorix kernel, i am using zfs with ubuntu 19.10
please add support it. thanks

**Comment by damentz** [comment]
Have you searched for any guides on using ZFS with Ubuntu? Ubuntu provides a ZFS DKMS package here: https://packages.ubuntu.com/search?keywords=zfs-dkms

**Comment by ghost** [comment]
the official kernel native support ,but the liquorix not work. 
(fresh install ubuntu 19.10 can choose zfs , so i have not do much working about it)

**Comment by damentz** [comment]
The real problem is that ZFS is technically not native to Ubuntu, yet.  They use a modified scripts to include out-of-tree ZFS modules with the build.  That means there's no configuration for it.

In order to get ZFS native in Liquorix, we would need a built-in patch added to https://github.com/zen-kernel/zen-kernel.  And even if we do that, having ZFS built in would probably conflict with distribution packages, since at this point, installing ZFS packages almost always requires the full suite (DKMS modules + userspace packages).

Your only option at the moment is to use DKMS.  I did test Liquorix with `zfs-dkms` on 19.10 and it fails to build.  That coincides with the version Ubuntu is supporting on 19.10, which is 0.8.1.  According to the changelog for 0.8.1, this version builds up through Linux 5.1.  Version 0.8.2 builds through to Linux 5.4.

Basically, you'll need to upgrade to Ubuntu Focal or backport the latest ZFS packages on Ubuntu to 19.10 in order to use ZFS with Liquorix on Ubuntu.

**Comment by ghost** [comment]
Hard for me do the works. I am not senior kernel developer, just normal user..

thanks for explain that.
